include/solvers/navier_stokes_base.h

```diff
@@ -376,11 +376,13 @@ NavierStokesBase<dim, VectorType, DofsType>::NavierStokesBase(
   , mpi_communicator(MPI_COMM_WORLD)
   , n_mpi_processes(Utilities::MPI::n_mpi_processes(mpi_communicator))
   , this_mpi_process(Utilities::MPI::this_mpi_process(mpi_communicator))
-  , triangulation(this->mpi_communicator,
-                  typename Triangulation<dim>::MeshSmoothing(
-                    Triangulation<dim>::smoothing_on_refinement |
-                    Triangulation<dim>::smoothing_on_coarsening))
-  , dof_handler(this->triangulation)
+  , triangulation(dynamic_cast<parallel::DistributedTriangulationBase<dim> *>(
```

Comment:
This is the place where an instance of `p:f:t` will be created.

Reply:
Seems coherent